README.md

```diff
@@ -107,3 +128,66 @@ You should be aware that setting this variable provides a safe *subset* of
 functionality; it is still possible that PRs raised for code repositories will
 still fail under the automated CI systems since those systems are running all
 possible tests.
+
+
+## Run the Kata Containers tests
+
+### Requirements to run Kata Containers tests
+
+You need to install the following to run Kata Containers tests:
+
+- [golang](https://golang.org/dl)
+
+  To view the versions of go known to work, see the `golang` entry in the
+  [versions database](https://github.com/kata-containers/runtime/blob/master/versions.yaml).
+
+- `make`.
+
+### Prepare an environment
+
+The recommended way to set up Kata Containers is using the official and latest
+stable release. You can find the official documentation to do this in the
+[Kata Containers installation user guides](https://github.com/kata-containers/documentation/blob/master/install/README.md).
+
+If you want to try latest commits of Kata you can use the CI scripts, which will build
```

Comment:
Lines 152-153 suggested rewrite: 
To try the latest commits of Kata use the CI scripts, which build and install from the kata-containers repositories, with the following steps: